bedrock/mozorg/templates/mozorg/about/governance/policies/security/bugs.html

```diff
@@ -12,10 +12,10 @@ <h1 class="title-banner">{{ _('Handling Mozilla Security Bugs') }}</h1>
 
```

Comment:
I think the h1 on this page should get `class="title-shadow-box"` for the fancy red box treatment (until we rebrand all these pages, anyway).